src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -51,6 +48,7 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfoWithFileRename
             public SymbolAndProjectId RenameSymbolAndProjectId { get; }
             public bool HasOverloads { get; }
             public bool ForceRenameOverloads { get; }
+            public IEnumerable<DocumentSpan> DocumentSpans { get; }
```

Comment:
Prefer ImmutableArray instead of IEnumerable for storing data.  it prevents accidental unexpected expensive lazyness.  (there are also concerns about IEnumerable and sharing across threads, but that's harder to explain over a github message).

Reply:
Consider calling this something like DefinitionLocations (and commenting) to clarify what this is.